lib/elixir/lib/kernel/cli.ex

```diff
@@ -287,36 +327,40 @@ defmodule Kernel.CLI do
   end
 
   defp parse_compiler(["-o", h | t], config) do
-    parse_compiler t, %{config | output: h}
+    parse_compiler(t, %{config | output: h})
   end
 
   defp parse_compiler(["--no-docs" | t], config) do
-    parse_compiler t, %{config | compiler_options: [{:docs, false} | config.compiler_options]}
+    parse_compiler(t, %{config | compiler_options: [{:docs, false} | config.compiler_options]})
   end
 
   defp parse_compiler(["--no-debug-info" | t], config) do
-    parse_compiler t, %{config | compiler_options: [{:debug_info, false} | config.compiler_options]}
+    updated_compiler_options = [{:debug_info, false} | config.compiler_options]
+    parse_compiler(t, %{config | compiler_options: updated_compiler_options})
```

Comment:
`updated_compiler_options` variable used to keep lines cleaner

Reply:
You can just call it `compiler_options`, here and below. :)